projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -803,24 +879,26 @@ static function( $matches ) {
 		);
 	}
 
-	function get_attachment_comments() {
+	/**
+	 * Allows for the checking of privileges of the blog user before comments
+	 * are packaged as JSON and sent back from the get_attachment_comments
+	 * AJAX endpoint
+	 *
+	 * @module carousel
+	 *
+	 * @since 1.6.0
+	 *
+	 * @return string
+	 */
+	public function get_attachment_comments() {
 		if ( ! headers_sent() ) {
 			header( 'Content-type: text/javascript' );
 		}
 
-		/**
-		 * Allows for the checking of privileges of the blog user before comments
-		 * are packaged as JSON and sent back from the get_attachment_comments
-		 * AJAX endpoint
-		 *
-		 * @module carousel
-		 *
-		 * @since 1.6.0
-		 */
 		do_action( 'jp_carousel_check_blog_user_privileges' );
 
-		$attachment_id = ( isset( $_REQUEST['id'] ) ) ? (int) $_REQUEST['id'] : 0;
-		$offset        = ( isset( $_REQUEST['offset'] ) ) ? (int) $_REQUEST['offset'] : 0;
+		$attachment_id = ( isset( $_REQUEST['id'] ) ) ? (int) $_REQUEST['id'] : 0; // phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Some sanitisation does exist here.
```

Comment:
Nonce check may not be needed - more conversation around this here: p1644583040300119-slack-C02HQGKMFJ8